### PR TITLE
chore: move processLauncher into src/utils

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: src/utils/eventsHelper.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      emitter: EventEmitter;
+      eventName: (string | symbol);
+      handler: (...args: any[]) => void;
+    }>) {
+    for (const listener of listeners)
+      listener.emitter.removeListener(listener.eventName, listener.handler);
+    listeners.splice(0, listeners.length);
+  }
+
+  static completeUserURL(urlString: string): string {
```
Comment (dgozman) [inline src/utils/eventsHelper.ts:45]: Accidental copy?

File: utils/check_deps.js [hunk 2]
```diff
@@ -148,7 +148,7 @@ DEPS['src/web/recorder/'] = ['src/common/', 'src/web/', 'src/web/components/', '
 DEPS['src/web/traceViewer/'] = ['src/common/', 'src/web/'];
 DEPS['src/web/traceViewer/ui/'] = ['src/common/', 'src/protocol/', 'src/web/traceViewer/', 'src/web/', 'src/server/trace/viewer/', 'src/server/trace/', 'src/server/trace/common/', 'src/server/snapshot/snapshotTypes.ts', 'src/protocol/channels.ts'];
 // The service is a cross-cutting feature, and so it depends on a bunch of things.
-DEPS['src/remote/'] = ['src/client/', 'src/debug/', 'src/dispatchers/', 'src/server/', 'src/server/supplements/', 'src/server/electron/', 'src/server/trace/'];
+DEPS['src/remote/'] = ['src/client/', 'src/debug/', 'src/dispatchers/', 'src/server/', 'src/server/supplements/', 'src/server/electron/', 'src/server/trace/', 'src/utils/processLauncher.ts'];
```
Comment (dgozman) [inline utils/check_deps.js:151]: Just make it `src/utils/**`

File: utils/check_deps.js [hunk 3]
```diff
@@ -161,7 +161,7 @@ DEPS['src/utils/'] = ['src/common/', 'src/protocol/'];
 DEPS['src/server/trace/common/'] = ['src/server/snapshot/', ...DEPS['src/server/']];
 DEPS['src/server/trace/recorder/'] = ['src/server/trace/common/', ...DEPS['src/server/trace/common/']];
 DEPS['src/server/trace/viewer/'] = ['src/server/trace/common/', 'src/server/trace/recorder/', 'src/server/chromium/', ...DEPS['src/server/trace/common/']];
-DEPS['src/test/'] = ['src/test/**', 'src/utils/utils.ts', 'src/server/processLauncher.ts'];
+DEPS['src/test/'] = ['src/test/**', 'src/utils/utils.ts', 'src/utils/processLauncher.ts'];
```
Comment (dgozman) [inline utils/check_deps.js:164]: Just make it `src/utils/**`

**Review by dgozman — APPROVED** [review]

File: src/utils/eventsHelper.ts (hunk 1 above)
Comment (mxschmitt) [reply src/utils/eventsHelper.ts]: good catch!

**Pull request merged**